code/style.css

```diff
@@ -1,7 +1,327 @@
+  height: 100vh;
+}
+ 
+.hero-image h1 { /* Text on hero-image and placement of it */
+  text-align: center;
+  margin: 5px;
+  color: #22031F;
+  text-shadow: 3px 3px #87B38D;
+  font-Family: 'Roboto', Sans-Serif;
+  font-Size: 106px;
+}
+
+.hero-image h3 { /* Text on hero-image and placement of it */
+  text-align: center;
+  margin: 5px;
+  color: #87B38D;
+  font-size: 24px;
+}
+
+main { /** The area containing the form and the FAQ. **/
+  display: flex;
+  justify-content: space-evenly;
+  padding: 20px;
+  border-top: 15px solid white;
+  height: 120vh;
+}
+
+/*** FORM ***/
+.form-container {
+  margin: 40px;
+  background-color: #ddeaeb;
+  padding: 10px 15px;
+  width: 550px;
+  max-height: 690px;
+  border-radius: 5%;
+  cursor: grab;
+}
+
+h2 {
+  font-Family: 'Roboto', Sans-Serif;
+}
+
+.form-container:hover { /* When hovering the full container-box */
+  box-shadow: 3px 3px 20px 5px #DD9296;
+}
+
+/* Flexbox for the form attributes */
+.form { 
+  display: flex;
+  flex-direction: column;
+}
+
+.form-div {
+  margin-bottom: 5px;
+  display: flex;
+  align-items: center;
+}
+
+.form-div label { /* Design of the labels */
+  width: 100px;
+  margin: 5px;
+}
+
+#fullName, #companyName, #mailAdress, #passWord {  /*Styling of the input fields*/
+  width: 250px;
+  border: none;
+  border-bottom: 1px solid black;
+  background-color: #ddeaeb;
+}
+
+.checkbox { /* Flexboxes for the checkboxes */
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+}
+
+.checkbox-child {
+  margin-bottom: 5px;
+}
+
+/** Design of the button **/
+button  { 
+  width: 438px;
+  font-size: 16px;
+  font-weight: bold;
+  background: #ddeaeb;
+  color: #22031F;
+  padding: 8px;
+  margin: 10px;
+  border: 1px solid white; 
+  box-shadow: 2px 2px 5px 2px white; 
+}
+
+button:hover { /** How the button shall look when pointing on it - it gets flat **/ 
+  transform: translateY(5px);
+  box-shadow: none;
+}
+
+/*** ACCORDION ***/
+.accordion {
+  margin: 40px;
+  background-color: #ddeaeb;
+  padding: 10px 15px;
+  display: flex;
+  flex-direction: column;
+  width: 550px;
+  max-height: 690px;
+  border-radius: 5%;
+  cursor: pointer;
+}
+
+.accordion-container-1 {
+  background-color: white;
+  padding-bottom: 20px;
+}
+
+.accordion-container-2 {
+  background-color: #e1e8ed;
+  padding-bottom: 20px;
+}
+
+.accordion:hover { /* When hovering */
+  box-shadow: 3px 3px 20px 5px #DD9296;
+}
+
+.question { /* Design of the question texts */
+  font-weight: bold;
+  line-height: 1em;
+  cursor: pointer;
+  transition: 0.3s;
+  margin: 10px;
+}
+
+.question:hover { /* When hovering the question 'heading */
+  font-size: 15px;
+}
+
+.question::after {
+  content: "\002B";
+  font-size: 24px;
+  float: left;
+  margin-right: 10px;
+}
+
+.question.active::after {
+  content: "\2212";
+}
+
+.question.remove::after {
+  content: "\002B";
+}
+
+.answer-container {
+  height: 0;
+  overflow: hidden;
+  transition: height 0.3s ease; 
+}
```

Comment:
I understand that you were trying to get a smooooooth transition in when you toggle your accordion. It's hard to understand and see exactly where the styling should go. If you check the Inspector when your clicking on a FAQ you'll see what's changing and where to move this styling - right now it doesn't listen to this styling. Try it out when you can and let me know if you want any guidance 👋

Reply:
@bealun Thank you! I would really like to get some guidance if possible - I do not really understand what is wrong and how to change it.. :/  If I remove this styling it doesn't work in the correct way.